### ISSUE TITLE
CRAN version lacks tests

### ISSUE THREAD

**Issue by barracuda156** [HenryLeongStat/regmhmm#1]
Tests dir is missing from CRAN tarball. Could that be fixed?

**Comment by HenryLeongStat** [comment]
@barracuda156 Yes! More tests in the `tests` folder created using the `testthat` package will be added in the next release. The current release actually covers most of the tests in the example sections of each function.

Thanks for creating the first issue!

**Comment by barracuda156** [comment]
@HenryLeongStat Got it, thank you!

I have added your package to Macports: https://ports.macports.org/port/R-regmhmm
(If you will be interested to be a maintainer for it, please let me know.)
